docs/tutorials/models.ipynb

```diff
@@ -51,7 +51,7 @@
    "outputs": [],
```

Comment:
<p>I think in <code>.evaluation_radius()</code>  we should choose the larger one like  <code>np.max(sigma_1TeV, sigma_10TeV)</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
Ok! Done